src/plugins/widgets/search/SearchSettings.tsx

```diff
@@ -32,6 +34,43 @@ const SearchSettings: React.StatelessComponent<Props> = ({
         )}
       </select>
     </label>
+
+    <label>
```

Comment:
If you wanted to get really fancy, you could hide this if the current search provider doesn't have a `suggest_url` set.

Reply:
Yeah, but the suggestions wouldn't work if I use for example duckduckgo, personally I would like to receive suggestions for duckduckgo as well.